types/parity-pmd/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for parity-pmd 0.4
+// Project: https://github.com/paritytrading/node-parity-pmd#readme
+// Definitions by: Leonard Vujanić <https://github.com/leovujanic>
```

Comment:
@jewbre can you please add your name here?
Here is a snippet
```
// Definitions by: Alice <https://github.com/alice>
//                          Bob <https://github.com/bob>
```

Reply:
Since it was your type, I didn't even think about that. :)
Added.